packages/dds/sequence/src/test/compare.py

```diff
@@ -0,0 +1,46 @@
+import json
```

Comment:
Did you mean to include these scripts?

Reply:
Ah must've slipped in during a merge conflict. Good catch!